src/database/queries.ts

```diff
@@ -100,3 +106,49 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Create a new "vendor" document and at it to the vendors collection. 
+ */
+ export async function createVendor(email: string, name: string) {
```

Comment:
The argument should use the type `Vendor` (defined in `types.tsx`).

Since we don't want all fields in Vendor to be required here (e.g. we won't have a uuid before actually making the firebase query), you can use `Pick` similar to how @sauhardjain did it in #10 

See docs on Pick: https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys
```
export type VoucherCreate = Pick<
  Voucher,
  'type' | 'value' | 'vendorUuid' | 'expiration_date'
>;
```